docs/program/apis/disconnect.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Session Connectivity: Network Disconnection during a Session"
+linkTitle: "Network Disconnection"
+weight: 20
+type: "docs"
+description: "How Viam handles losing connection to the internet during a robot session."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session"]
+---
+
+What happens when your robot loses its connection to the internet over a LAN or WAN and can no longer communicate through [the Viam app](https://app.viam.com).
+
+### How does `viam-server` work if your robot loses connectivity during a session?
+
+If the robot loses connectivity, `viam-server` will timeout and end any current client [*sessions*](/program/apis/sessions/) on this robot.
+When its session times out, each client is asked to cancel any ongoing operations by the automatically-configured `SessionsClient` of your robot. 
+Any active commands will be cancelled, stopping any moving parts, and no new commands will be able to reach the robot until the connection is restored.
+
+### How will your robot's primary parts and sub parts communicate if internet connectivity is interrupted?
+
+Resources on the robot will still be able to send operations requests to each other if your robot running `viam-server` is no longer connected to the internet and the session is ended, but all operations will timeout automatically and cancel if network connectivity is not re-established under the management of the `SessionsClient`.
```

Comment:
@Otterverse reworked line 20 a lot but I'm not sure about the immediacy of session end-- is it more accurate currently or is the below suggestion correct? 

```suggestion
Resources on the robot will still be able to send operations requests to each other if your robot running `viam-server` is no longer connected to the internet and the session is ended, but all operations will timeout automatically and operations will be halted.
```